charter.html

```diff
@@ -158,14 +158,15 @@ <h2>Scope</h2>
             be implemented in browsers or similar user agents. The purpose of
             these features is to support web advertising without compromising
             user privacy.  Here "privacy" minimally refers to appropriate
-            processing of personal information.  Ways in which new features might
-            enable inappropriate processing include (but are not limited to)
-            enabling
+            processing of personal information that is collected by a user agent.  
```

Comment:
I'm confused: what is the new clause "that is collected by a user agent" supposed to accomplish here?  It seems like it would unnecessarily restrict the scope of the WG.

For example, one might argue that the aggregate conversion measurement work in progress in PATCG is _out of scope of the WG_ based on this clause, because the way conversion tracking works today does _not_ involve information being collected _by a user agent._

Reply:
Agreed with @michaelkleber's concern. Would saying "collected or created by a user agent" fix the issue?